Fastfile

```diff
@@ -81,7 +58,8 @@ platform :ios do
   end
 
   desc "Releases a new version to `TestFlight`. This lane must receive the following parameters:"
-  desc "- build_configuration: A build configuration to deploy. Can be any of: `%s`" % build_configurations.keys.to_s
+  desc "- build_configuration: A build configuration to deploy. \
+Can be any of: `%s`" % Actions::GetBuildConfigurationAction::BUILD_CONFIGURATIONS.keys.to_s
```

Comment:
I feel there's a space or something missing here

Reply:
No, this is the right way to do it. I don't like either the way it looks in the code, but the documentation is properly generated.